files/rman_backup_restore.sh

```diff
@@ -532,6 +531,14 @@ restore_db_passwords () {
     fi
     SSMNAME="/${HMPPS_ENVIRONMENT}/${APPLICATION}/${PRODUCT}-database/db/${SUFFIX}"
     USERPASS=`aws ssm get-parameters --region ${REGION} --with-decryption --name ${SSMNAME} | jq -r '.Parameters[].Value'`
+    if [ -z ${USERPASS} ]
```

Comment:
Should add an explanation of why this is here and a TODO to remove when the password parameter store keys are updated.

Reply:
I have made a note regarding this

Reply:
Did it myself. Needs to be in the code.